src/index.js

```diff
@@ -1,6 +1,9 @@
 import 'core-js/stable'
 import 'regenerator-runtime/runtime'
 
+// Initialize Sentry immediately if enabled
+import './sentry'
```

Comment:
Do we feel like doing this is a bit sketchy / hidden? Would it be better if it were done directly in the index.js instead?

Reply:
What about exporting a `initSentry()` function that would get called from here? Or is it to capture errors on import for `react`, `react-dom` and `@aragon/ui`?

Reply:
No, no good reason :).

I was going back and forth a couple times on it, but agree with you guys that mangling the global space on an import like this is probably less of a good idea.